### ISSUE TITLE
Is the measure class actually needed?

### ISSUE THREAD

**Issue by GordianDziwis** [HajoRijgersberg/OM#67]
Because of the Measure class it takes one more jump to get from the phenomenon to the numericalValue, this makes it more difficult to query data mapped to om. Wouldn't it be enough to just link the numericalValue directly to the quantity and get rid of the Measure class, like this:
![image](https://user-images.githubusercontent.com/3330855/172441598-dac66239-5d6c-4a37-a8e7-3f6ab57ab601.png)

**Comment by HajoRijgersberg** [comment]
No, because om:hasUnit lists all the possible units for a quantity, and one and the same quantity (instance) could have different values expressed with different units. So that is why - unfortunately - we need something like om:Measure.
My hope and ideal would be that in the future there will number datatypes with units exist. Not only in ontologies or OM, but in IT in general.

**Comment by HajoRijgersberg** [comment]
B.t.w., I fully agree with you that ease of querying, so to say, _is_ an important criterion, which I am giving ever more importance in the design.

**Comment by HajoRijgersberg** [comment]
For OM 3.0 I am investigating whether these custom number-unit datatypes could be constructed.

**Comment by HajoRijgersberg** [comment]
Update: I'm thinking more an more to get rid of the measure, as you have proposed, for querying ease and because presently no number - unit-symbol datatypes exist and defining these would also have disadvantages as these will not be commonly known (yet). In fact, this would be an entire different issue, a very important issue i.m.h.o., much much broader than OM only, but important worldwide, in a majority of IT applications, I expect.

The change would bring us to a crucial philosophical question: can or should a quantity only have one measure? And perhaps I already know the answer: yes, because each of these values may have a different acquirement process, provenance, origin.

**Comment by HajoRijgersberg** [comment]
The possible units for a quantity should in principle be expressed using class restrictions I think, not using multiple hasUnit relations as it is now and also not using commonlyHasUnit relations, which is an implicit way of restricting.
If we adopt your proposal, such class restrictions become possible again.
